examples/cli/src/main/java/io/unitycatalog/cli/utils/PythonInvoker.java

```diff
@@ -27,17 +27,21 @@ public static String invokePython(FunctionInfo function, String scriptPath, Stri
             argsList.add(function.getRoutineDefinition());
 
             // Retrieve and add parameters as arguments
-            List<FunctionParameterInfo> parameters = function.getInputParams()
+            List<FunctionParameterInfo> parameters = function
+                    .getInputParams()
                     .getParameters();
             if (parameters == null || parameters.isEmpty()) {
                 throw new ApiException("Function parameters not found.");
             }
+            if (args.length < parameters.size()) {
```

Comment:
Should we add a more descriptive message like which parameters were not provided?